src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/PrisonerIndexResourceTest.kt

```diff
@@ -35,7 +34,7 @@ class PrisonerIndexResourceTest : QueueIntegrationTest() {
     )
     resetStubs()
     setupIndexes()
-    Mockito.reset(indexQueueService)
+    // Mockito.reset(indexQueueService)
```

Comment:
I'm surprised this isn't needed as I thought all SpyBeans were not reset automatically.